content/docs/reference/addon-manifest.md

```diff
@@ -365,7 +365,7 @@ Specify a `settings` property and provide an array of option objects.
 Sub-properties:
 - `name` (string, required) The user-visible text for the option.   
 - `id` (string, required) An identifier to get the user-specified value from your code.  
-- `type` (string, required) Either `boolean` (an on/off toggle), `positive_integer` (an input box that only allows 0 and above), `integer` (an input box that allows any integer) `string` (up to 100 chars),`color` (a browser color input that returns a hex code), `table` (list of elements, where user can add custom elements, remove existing ones and change order of them) or `select` (see `potential_values`).  
+- `type` (string, required) Either `boolean` (an on/off toggle), `positive_integer` (an input box that only allows 0 and above), `integer` (an input box that allows any integer) `string` (up to 100 chars), `untranslated` (a string that isn't send for translation), `color` (a browser color input that returns a hex code), `table` (list of elements, where user can add custom elements, remove existing ones and change order of them) or `select` (see `potential_values`).

```

Comment:
Would be a nice opportunity to turn this into an item list.

I think the best way to describe the untranslated type is:
> Works as a string setting, but its default value is not sent to the extension translation process. For example, if the default value is a URL or a number.

Reply:
I started trying to turn it into a table locally but I'm not really sure exactly how to lay it out.